pkgs/by-name/gi/gitea/package.nix

```diff
@@ -88,7 +88,7 @@ in buildGoModule rec {
   '';
 
   passthru = {
-    data-compressed = lib.warn "gitea.passthru.data-compressed is deprecated. Use \"compressDrvWeb gitea.data\"." (compressDrvWeb gitea.data);
+    data-compressed = lib.warn "gitea.passthru.data-compressed is deprecated. Use \"compressDrvWeb gitea.data\"." (compressDrvWeb gitea.data {});
```

Comment:
I think you can just remove this trace. It's fine to go through either door: you can call the function yourself, or you can rely on the gitea derivation to provide it.

It _is_ a shame that `buildGoModule` doesn't support `(finalAttrs:` style, which would let this actually use the fixpoint finalPackage.

Reply:
Thanks! I'll see what I can do to get that merged.